persistence-cassandra-4.0/src/main/java/io/stargate/db/cassandra/impl/Conversion.java

```diff
@@ -369,48 +360,31 @@ public static Result toResult(
         return new Result.SetKeyspace(((ResultMessage.SetKeyspace) resultMessage).keyspace);
       case SCHEMA_CHANGE:
         return new Result.SchemaChange(toSchemaChangeMetadata(resultMessage));
+      case PREPARED:
```

Comment:
Was this breaking something since it was missing?

Reply:
No. There was a separate method to handle prepared results previously. As the code is more factored now, this has to move here.